### PR TITLE
Update how to ask questions / file bugs

### PR DESCRIPTION
**Pull request by mlvandijk**
This PR aims to clarify where to ask questions and how to file bug reports a little more.

### CODE REVIEW

**Review by olleolleolle — APPROVED** [review]
These wording changes nudge a reader's understanding in important ways: _which_ issue tracker is used, _what are the channels called_ and so on.

**Comment by mlvandijk** [comment]
Yes, that's what I'm aiming for. Thanks @olleolleolle

File: content/community/contributing.md [hunk 1]
```diff
@@ -51,14 +54,16 @@ The only way to fix a bug, is to reproduce it first. At the very least, provide
 If you can, write a Unit Test or a Cucumber Feature that demonstrates the bug.
 On occasion, we might ask you to provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) (preferably in the form of a Github project).
 
-If you file a Github issue without providing following these quidelines, your ticket may be closed. If that happens, please provide the missing information in the ticket to have it reopened.
+If you file a Github issue without providing following these guidelines, your ticket may be closed. If that happens, please provide the missing information in the ticket to have it reopened.
```
Comment (xtrasimplicity) [inline content/community/contributing.md:57]: Maybe we should also remove "providing"? `...without providing following these guidelines... `

Comment (mlvandijk) [reply content/community/contributing.md:57]: Nice catch! Will do

**Pull request merged**